CONTRIBUTING.md

```diff
@@ -41,6 +41,15 @@ When creating a new pull request:
 * If you decide to review a pull request with the intent of merging it (or deciding what still needs to be done before merge), then **assign the PR to yourself** so that it's clear that someone's grabbed it.
 * Anyone is welcome to informally review a PR and comment on it at any time, no matter who is assigned.
 
+## Team Workflow
+
+1. Pull an issue from the current sprint only plz (unless there is an *URGENT* issue). Take an unassigned issue or re-assign an issue with the label "[stealable](https://github.com/18F/openFEC/labels/stealable)." If a non *URGENT* issue comes up, but you want to prioritize it, coordinate with the PM or tech lead on slack.
```

Comment:
Small nitpick, but can we say "from the current sprint only, please", please? ;)